### DISCUSSION TITLE
Is the CLI being abandoned?

### DISCUSSION THREAD

**Discussion by alexbujenita** [sequelize/sequelize#13171] [Help & Questions] [answered]
Hi all,

I remember that once https://github.com/sequelize/cli used to have the Issues tab available, now there that has disappeared for some reason and also it doesn't have a discussion tab.

Is the project getting abandoned?

It was real useful to see some examples in the issues, especially while creating models and/or relations.

**Comment by papb** [comment, marked as answer]
Hello!! Thank you for this post. I am very sorry. I have just re-enabled issues there, and you will be able to find the old ones. I should have done this sooner. There's a lot of work to be done there as well. I will look into it this month.

It is not getting abandoned.

**Reply by alexbujenita** [reply]
Good news, thank you!
